packages/renderer/src/dom/PDFViewer.js

```diff
@@ -29,6 +29,7 @@ export const PDFViewer = ({
       ref={innerRef}
       style={style}
       className={className}
+      key={crypto.randomUUID()}
```

Comment:
Why does it need to be here and not added by consumer? Not everyone uses react-router to see this issue. This could generate unwanted rerenders for other people right?